examples/data/mf6-freyberg/top.asc

```diff
@@ -1,46 +1,46 @@
-29.3103 29.2280 29.0875 28.8364 28.3314 0.0000 24.7932 24.3044 23.5913 22.8861 21.9998 20.9296 19.5816 18.2895 17.1477 17.7552 18.2569 18.6048 18.7897 18.8824
-28.9468 28.8426 28.6508 28.3193 27.6730 26.5872 25.4529 24.6704 23.8975 23.1000 22.1186 20.9567 19.5528 18.1638 16.9189 17.5871 18.1062 18.4429 18.6478 18.7446
-28.6005 28.4909 28.2850 27.9009 27.2529 26.4791 25.6899 24.9284 24.0725 23.1880 22.1955 20.9565 19.4865 18.0044 16.6920 17.3869 17.9612 18.3042 18.5158 18.6142
-28.2770 28.1750 27.9555 27.5577 27.0176 26.4167 25.7344 25.0141 24.2137 23.3078 22.1882 20.8180 19.3577 17.8950 16.4772 17.2243 17.8237 18.1775 18.3985 18.5022
-27.9916 27.8798 27.6697 27.3405 26.8805 26.3494 25.7719 25.0419 24.2356 23.4149 22.0457 20.5413 19.3147 17.8586 16.2375 17.0741 17.7004 18.0803 18.3046 18.4042
-27.7348 27.6162 27.4340 27.1352 26.7672 26.3594 25.7994 24.9772 24.0951 23.1671 21.8350 20.4809 19.2232 17.7493 16.0085 16.9109 17.5897 17.9922 18.2116 18.3159
-27.4867 27.3801 27.2089 26.9490 26.6685 26.3594 25.9177 24.9173 23.7655 22.9540 21.6870 20.2960 19.0952 17.5830 15.7857 16.7455 17.5035 17.9127 18.1326 18.2280
-27.2659 27.1733 26.9982 26.7738 26.5477 26.3239 25.7956 24.7884 23.6624 22.6306 21.3745 20.0279 18.8725 17.4217 15.5584 16.5602 17.3754 17.8078 18.0255 18.1201
-27.0992 26.9894 26.7944 26.5460 26.3146 26.0418 25.5067 24.4936 23.4583 22.4206 21.1382 19.8711 18.6805 17.3357 15.3213 16.3865 17.1958 17.6652 17.8731 17.9495
-26.9855 26.7856 26.5277 26.2267 25.9753 25.6768 25.0787 24.1456 23.3126 22.2451 21.0996 19.7447 18.4381 17.1423 15.0914 16.1166 16.8945 17.3769 17.6306 17.7026
-0.0000 26.5581 26.2421 25.8987 25.5629 25.1570 24.5664 24.1068 23.2378 21.9430 21.0280 19.7403 18.0513 16.4333 14.8729 15.6985 16.5013 17.0384 17.3343 17.4400
-0.0000 0.0000 25.9003 25.5183 25.1175 24.6674 24.1454 23.6011 22.7846 21.9279 20.9036 19.5506 17.6990 16.0786 14.6397 15.5185 16.2272 16.7720 17.0695 17.2266
-0.0000 0.0000 0.0000 25.0825 24.5310 24.0663 23.5984 23.1007 22.4714 21.6958 20.5893 19.0353 17.2143 15.8583 14.4106 15.3139 15.9899 16.5181 16.8330 17.0850
-0.0000 0.0000 0.0000 0.0000 23.7381 23.3648 22.9378 22.4520 21.9098 21.1388 19.9832 18.6560 17.0354 15.5765 14.1908 15.1187 15.7983 16.2660 16.5142 0.0000
-0.0000 0.0000 0.0000 0.0000 23.2295 22.5662 22.0596 21.5957 21.1462 20.4836 19.4587 18.3429 16.9059 15.3844 13.9596 14.9805 15.6024 16.0389 16.3466 0.0000
-0.0000 0.0000 0.0000 0.0000 0.0000 21.2474 20.9537 20.6163 20.3183 19.7002 18.7647 17.8460 16.6769 15.2479 13.7210 14.8247 15.3862 15.8014 0.0000 0.0000
-0.0000 0.0000 0.0000 0.0000 0.0000 20.0273 19.8268 19.5266 19.3028 18.5765 17.9346 17.1333 16.1436 14.9701 13.5055 14.6651 15.1257 0.0000 0.0000 0.0000
-0.0000 0.0000 0.0000 0.0000 0.0000 18.9000 18.4000 18.1000 17.6000 17.1000 17.1000 16.0000 15.0000 14.5000 14.0000 0.0000 0.0000 0.0000 0.0000 0.0000
+NCOLS 20

```

Comment:
line ending changes?

Reply:
Yes. I'm having trouble reverting these, but will put in a new PR without these changes. These files are written to examples directory during the autotesting, which I'll examine and update so this doesn't continue to happen.